### PR TITLE
Governance links section

### PR DESCRIPTION
**Pull request by bobo-k2**
**Pull Request Summary**

Added
- governance link section to the portal
- governance link to the sidebar

This is currently available on Shibuya
image

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

### CODE REVIEW

File: src/components/governance/GovernanceLink.vue [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="link-wrapper">
+    <a :href="url" target="_blank" rel="noopener noreferrer">{{ title }}</a>
+    <div><astar-icon-arrow-right /></div>
```
Comment (gtg7784) [inline src/components/governance/GovernanceLink.vue:4]: Why we need to wrap icon with div?

File: src/components/sidenav/SidebarDesktop.vue [hunk 2]
```diff
@@ -126,12 +118,12 @@ export default defineComponent({
   setup() {
     const { isOpen } = useSidebar();
     const { isZkEvm, isZkyoto } = useNetworkInfo();
+    const { isEnabled: isGovernanceEnabled, homepageUrl } = useGovernance();
```
Comment (gtg7784) [inline src/components/sidenav/SidebarDesktop.vue:121]: imo, it's better to use `governanceUrl`

File: src/hooks/useGovernance.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  state: string;
+  url: string;
+};
+
+const proposals = ref<GovernanceData[]>([]);
+const ongoingReferenda = ref<GovernanceData>();
+
+const fetchProposals = async (network: string): Promise<GovernanceData[]> => {
+  try {
+    const url = `https://${network}.subsquare.io/api/democracy/proposals?simple=true&page=1&page_size=2`;
```
Comment (gtg7784) [inline src/hooks/useGovernance.ts:17]: is this page size is because we need to fetch only two proposals?

Comment (bobo-k2) [reply src/hooks/useGovernance.ts:17]: Yes, it is because of that

File: src/components/governance/GovernanceLink.vue (hunk 1 above)
Comment (bobo-k2) [reply src/components/governance/GovernanceLink.vue:4]: I am using this as solution with the issue I had when arrow icons are resized based on link text size
image

File: src/components/sidenav/SidebarDesktop.vue (hunk 2 above)
Comment (bobo-k2) [reply src/components/sidenav/SidebarDesktop.vue]: Fixed in https://github.com/AstarNetwork/astar-apps/pull/1402/commits/4d7e1095910a053bf8ffe8d3e9a851284ee42368

**Comment by Kahonnohak** [comment]
hm.. my preview is not showing anything?
Screenshot 2024-10-22 at 09 18 25
Screenshot 2024-10-22 at 09 18 17

**Comment by Kahonnohak** [comment]
Wow great! This really gets attention:) 
- Can you add dark theme please
- Can we say maximum 3 lines and if its longer, would you add "..." ?
Screenshot 2024-10-22 at 09 20 25

**Comment by Kahonnohak** [comment]
Please also add Governance to the mobile menu

**Review by gtg7784 — APPROVED** [review]

**Review by impelcrypto — APPROVED** [review]

**Pull request merged**